### PR TITLE
Set MQTT and XMPP clients to connect after server start up

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java [hunk 1]
```diff
@@ -50,34 +58,41 @@
...
-    public static final String MQTT_PROTOCOL = "MQTT";
-
     private ArduinoMQTTConnector arduinoMQTTConnector;
-    private static Map<String, LinkedList<String>> replyMsgQueue = new HashMap<>();
-    private static Map<String, LinkedList<String>> internalControlsQueue = new HashMap<>();
     private ConcurrentHashMap<String, String> deviceToIpMap = new ConcurrentHashMap<>();
 
     /**
-     * @param arduinoMQTTConnector an object of type "ArduinoMQTTConnector" specific for this ArduinoControllerService
+     * @return the queue containing all the MQTT reply messages from all Arduinos communicating to this service
```
Comment (harshanL) [inline components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java]: Put a proper comment with formatting.

File: components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java [hunk 2]
```diff
@@ -50,34 +58,41 @@
...
-        } else {
-            log.warn("MQTT disabled in 'devicemgt-config.xml'. Hence, ArduinoMQTTConnector not started.");
+    public static Map<String, LinkedList<String>> getReplyMsgQueue() {
+        return replyMsgQueue;
+    }
+
+    /**
+     * @return the queue containing all the MQTT controls received to be sent to any Arduinos connected to this server
+     */
+    public static Map<String, LinkedList<String>> getInternalControlsQueue() {
```
Comment (harshanL) [inline components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java]: Put a proper comment with formatting.

File: components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java (hunk 1 above)
Comment (charithag) [inline components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java]: Fix with PR https://github.com/wso2/carbon-device-mgt-plugins/pull/206

File: components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java (hunk 2 above)
Comment (charithag) [inline components/device-mgt-iot-arduino/org.wso2.carbon.device.mgt.iot.arduino.controller.service.impl/src/main/java/org/wso2/carbon/device/mgt/iot/arduino/controller/service/impl/ArduinoControllerService.java]: Fix with PR https://github.com/wso2/carbon-device-mgt-plugins/pull/206

**Pull request merged**